docs/reference/upgrading-ehrql-from-v0-to-v1.md

```diff
@@ -0,0 +1,123 @@
+This page is a reference for ehrQL users
+that have existing dataset definitions
+written for versions of ehrQL labelled as v0.
+
+With the increment of major version from v0 to v1,
+ehrQL is now considered a suitable replacement for cohort-extractor.
+
+The v1 release also includes changes to ehrQL
+that may require some updates to existing dataset definitions using ehrQL v0.
+These changes are [explained below](#breaking-changes).
+
+## General notes
+
+### ehrQL replaces cohort-extractor for new projects
+
+* Newly approved OpenSAFELY projects now use ehrQL,
+  and not cohort-extractor.
+  Existing projects using cohort-extractor can continue to use cohort-extractor.
+* Earlier versions of the ehrQL software were known as "Data Builder".
+  ehrQL is now both the name of the electronic health record query language,
+  and the software that runs queries written in the ehrQL language.
+
+### Updates to documentation
+
+* The OpenSAFELY documentation has been updated to use ehrQL,
+  instead of cohort-extractor.
+* [Documentation describing the table schemas](schemas.md) has been considerably expanded.
+
+## Breaking changes
+
+!!! warning
+    This section describes ehrQL changes in v1.
+    Existing ehrQL v0 projects may need to make some or all of these changes
+    to use ehrQL v1.
+
+### Using the ehrQL action
+
+* If you were using the `ehrql:v0` action in `project.yaml`,
+  update from `ehrql:v0` to `ehrql:v1` in your `project.yaml`.
+* New versions of the `databuilder` Docker image
+  used by the `databuilder` action in `project.yaml`
+  are no longer published.
+  Update references such as `databuilder:v0` in your `project.yaml` to
+  `ehrql:v1`
+
+### Removal of `databuilder` imports
+
+* Update lines in your dataset definition that use `databuilder` imports to `ehrql`.
+  The `databuilder` imports are removed in ehrQL v1.
+  For example:
+
+  ```python
+  from databuilder.tables.beta.core import clinical_events
+  …
+  ```
+
+  is now:
+
+  ```python
+  from ehrql.tables.beta.tpp import clinical_events
+  ```
+
+### Renaming of raw tables
+
+* The "raw" minimally transformed tables are now imported
+  with names starting `beta.raw`.
+  Some tables have also been renamed to remove `_raw` from their name.
```

Comment:
There shouldn't be any tables with this prefix. Although there are several classes with the prefix, they aren't decorated with the `table` decorator. When they are passed to this function, they are named without the prefix. That's a mouthful; an example is clearer. Compare `ehrql.tables.beta.raw.tpp.isaric_raw` with `ehrql.tables.beta.raw.tpp.isaric`.